{
  error: "Une erreur est survenue",
  error_database: "Une erreur interne est survenue",
}
